python/llm/test/benchmark/arc-perf-test.yaml

```diff
@@ -27,3 +27,6 @@ in_out_pairs:
 test_api:
   - "transformer_int4_gpu"  # on Intel GPU
 cpu_embedding: False # whether put embedding to CPU (only avaiable now for gpu win related test_api)
+exclude:
+  - 'baichuan-inc/Baichuan2-13B-Chat:1024'
+  - 'baichuan-inc/Baichuan2-13B-Chat:2048'
```

Comment:
Baichuan2-13B is not here actually, you should first exclude 1k and 2k of moss?

Reply:
Sure